grails-app/services/au/org/emii/aatams/PermissionUtilsService.groovy

```diff
@@ -21,6 +21,7 @@ class PermissionUtilsService
 
     private static final Map<Long, String> PROJECT_READ = [:]
     private static final Map<Long, String> PROJECT_WRITE = [:]
+    private static final Map<Long, String> PROJECT_EDIT = [:]
```

Comment:
It may be a bit confusing as to the difference between `PROJECT_EDIT` and `PROJECT_WRITE`?